worlds/terraria/docs/setup_en.md

```diff
@@ -3,20 +3,32 @@
 ## Required Software
 
 Download and install [Terraria](https://store.steampowered.com/app/105600/Terraria/)
-and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Steam
+and [tModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Steam
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open tModLoader
+3. Go to "Workshop" -> "Manage Mods" and enable the Archipelago mod
+   - If tModLoader states that you need version 1.4.3, follow the following steps
+     1. Close tModLoader
+     2. Right-Click tModLoader in Steam and select "Properties"
+     3. Navigate to "Betas" -> "Beta Participation"
+     4. Select "1.4.3-legacy - Legacy - Stable tModLoader for Terraria 1.4.3"
+     5. Update tModLoader through Steam
+     6. Open tModLoader and navigate back to the **Manage Mods** menu
+4. tModLoader will say that it needs to refresh; exit this menu, and it will do this automatically
+5. Once tModLoader finishes loading, the Archipelago mod is finished installing; you can now 
+[connect to an Archipelago game](#joining-an-archipelago-game-in-terraria).
 
-This mod might not work with mods that significantly alter progression or vanilla features. It is
+This mod may not work with mods that significantly alter progression or vanilla features. It is
 highly recommended to use utility mods and features to speed up gameplay, such as:
 
 - Journey Mode
 - Ore Excavator
 - Magic Storage
 - Alchemist NPC Lite
-    - (May be used to break progression)
+    - (Can be used to break progression)
```

Comment:
While I think this was intended as another voice change, you have technically corrected grammar.
"May" indicates permission while "Can" indicates possibility.

Reply:
In this particular case, it was intentional